policyengine_us/variables/gov/states/de/tax/income/credits/eitc/refundability_calculation/de_tax_unit_eitc_refundable.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class de_tax_unit_eitc_refundable(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Delaware refundable earned income tax credit"
+    unit = USD
+    documentation = "Whether tax unit selects the refundable or non-refundable earned income tax credit."
+    definition_period = YEAR
```

Comment:
```suggestion
    definition_period = YEAR
    defined_for = StateCode.DE
```

Reply:
Though, given the extra computation needed for branching, we may be able to optimize further by defining a variable first indicating whether they might benefit from either EITC. That would be whether federal EITC > 0 right?